.TAOS-CI/.gbs.conf

```diff
@@ -27,10 +27,10 @@ buildroot = ~/GBS-ROOT-Snapshot/
 url = https://api.tizen.org
 
 [repo.base]
-url = http://download.tizen.org/snapshots/tizen/base/latest/repos/standard/packages/
+url = http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/latest/repos/standard/packages/
```

Comment:
LGTM

Q: ```https://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base/latest/repos/standard/packages/``` does also work. Is it possible to use HTTPS URL here?

Reply:
I do not know either. We received a guide with HTTP. Later we may change it to https.